blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+for monitoring SPI flash chips, and efforts toward stateless laptops, which
+eliminate sensitive data persistence. The talk outlined a retrofit plan to
+enhance hardware security, particularly within the Qubes OS ecosystem.
+
+Michał Żygowski talked about [Anti Evil Maid status and future plans](https://www.youtube.com/watch?v=5ieNhbLLTIU).
+AEM is a security tool designed to protect against attacks that tamper with the
+boot process of systems like Qubes OS. Michał presented an interesting case on
+the performance of AEM on NovaCustom NV4x Alderlake laptop and the results he
+achieved.
+
+Marek Marczykowski-Górecki and Frédéric Pierret presented an [Update on Qubes Air](https://www.youtube.com/watch?v=V4flhwEITr4),
+indicating the current issues, for example focus on Qrexec calls to other
+systems anddisposable qube support.
+
+Moreover, in this version, there is no GUI support for remote qubes, focus is on
+individual VMS on both systems and not on automatically synchronizing all the
+cube machines to the other. There is also a new type of Qube – a Relay Qube – a
+specialized LocalVM or a RemoteVM that acts as a bridge between the local and
+remote Qubes OS hosts.
+
+At the end of day one, there was a Qubes OS Summit 2024 – Day 1 closing notes
+hosted by Piotr Król who invited all conference participiants to the afterparty
+at Sudblock.
+
+## Day 2
+
+Day two of the Qubes OS Summit 2024 began with a brief
+[Welcome to Qubes OS Summit Day 2](https://www.youtube.com/watch?v=9AkBeBwxdA0)
+by Piotr Król, who welcomed attendees, once again acknowledged
+conference sponsors, and provided an overview of the second day’s agenda.
+
+The first talk of the second day was
+[NovaCustom: introducing the new Qubes OS certified V54 and V56 Series](https://www.youtube.com/watch?v=RV-1IR_d1Gg),
+hosted by Wessel klein Snakenborg and Tijn Veldhuis. They present a roadmap for
+the next few months, including, for example, developing comprehensive firmware
+updates and the UEFI updates.
+
+Tijn announced the new laptop model V56 and its specifications, including up to
+96 GB of internal memory. Wessel mentioned that they are also planning a Dasharo
+ACPI driver for the laptops, which provides communication between the OS and
+firmware.
+
+The following presentation,
+[Implementing UEFI Secure Boot in Qubes OS: Challenges and Future Steps](https://www.youtube.com/watch?v=ZcF_RN04oq8),
+was held by Piotr Król. Piotr started the talk by pointing out what UEFI Secure
+Boot is and what led the open-source software to dislike it. This talk explored
+the challenges and potential solutions for implementing UEFI Secure Boot in
+Qubes OS. An essential part of this talk was the presentation of future steps
+toward full Secure Boot support and how the community can participate in ongoing
+testing, feedback, and development work. Without a doubt, the first step that
+Secure Boot users can consider is choosing hardware that has better UEFI Secure
+Boot.
```

Comment:
I would remove the last sentence.

Reply:
The sentence has been removed as per the suggestion.